src/uuid/mod.rs

```diff
@@ -509,6 +509,30 @@ macro_rules! trait_impls {
                 uuid.serialize(serializer)
             }
         }
+
+        #[cfg(all($feat, feature = "serde_with3"))]
```

Comment:
I think this (and below) need to be `feature = "serde_with-3"` to match Cargo.toml.